license_sh/analyze/maven.py

```diff
@@ -274,6 +274,8 @@ async def fetch_concurrent():
                     output, dep_id = await result
                 except aiohttp.client_exceptions.ClientConnectorCertificateError:
                     continue
+                except UnicodeDecodeError:
```

Comment:
Can you please elaborate more on why this specific exception (and no others) should be caught?

Reply:
As much as possible but still we should not disable all

Reply:
Let's check the mimetype instead of catching an exception please.

Reply:
Ideally, also add an explanation comment. Let's avoid cryptic pieces of code.